Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM adoptopenjdk/openjdk11
```

Comment:
nie lepiej samo jre?

Reply:
nwm zawsze wszedzie bylo jdk tam gdzie zwiedzalem

Reply:
imo w tym przypadku wystarzczy jre chyba ze zrobimy tym "multi stagem" o ktorym mowiles to wtedy to jdk sie przyda, mozna zostawic